README.md

```diff
@@ -55,31 +92,42 @@ This would shutdown the system.
 - Run tests with test coverage info: `yarn test:cover`
 - Check the codebase for proper syntax and formatting compliance: `yarn lint`
 - Run your app in local dev mode: `yarn dev`. This puts the bundled app in a `dist` folder, set up a local web server at localhost:5000, and continues to watch for your code changes which it syncs with the local server. This means if you loaded the app in a browser, it will auto-refresh as you code along. Feel free to use whatever bundler best meets your needs. Parcel was only added as a sample and for those looking for a simple but effective solution to the hassle of bundlers. 
-- You can access the REST API interface from localhost:5000/api/<path>
-- The paths are /profile /profile/id etc.
+- You can access the REST API interface from [localhost:5000/api/](https://teachers-placement-backend.herokuapp.com)
+- The paths are:
+- `/health`
+- `/teachers`
+- `/teachers/me`
+- `/teachers/approved`
+- `/login`
 
 ## Authors
 
 - Ifere Okibe [LinkedIn](https://www.linkedin.com/in/ifere-jo/) [Twitter](https://twitter.com/Iam_Ifere)
 - Opysko
 - Shedrack Ajaegbu [LinkedIn](https://www.linkedin.com/in/ajaegbu-shedrack/) [Twitter](https://twitter.com/theshedman_)
-- Samuel Alex
+- Samuel Alexzender [linkedIn](https://www.linkedin.com/in/abiokorigho-samuel/) [Twitter](https://twitter.com/Alexzendersamu4)
 
 ## Contributing
 If this project sounds interesting to you and you'd like to contribute, thank you!
 First, you can send a mail to buildforsdg@andela.com to indicate your interest, why you'd like to support and what forms of support you can bring to the table, but here are areas we think we'd need the most help in this project :
-1.  area one (e.g this app is about human trafficking and you need feedback on your roadmap and feature list from the private sector / NGOs)
-2.  area two (e.g you want people to opt-in and try using your staging app at staging.project-name.com and report any bugs via a form)
-3.  area three (e.g here is the zoom link to our end-of sprint webinar, join and provide feedback as a stakeholder if you can)
+1.  Area two: We want people to opt-in and try using staging app at staging.project-name.com and report any bugs via a form.
+2.  Area three: here is the [zoom-link] to our end-of sprint webinar, join and provide feedback as a stakeholder if you can
 
 ## Acknowledgements
 
-Did you use someone else’s code?
-Do you want to thank someone explicitly?
-Did someone’s blog post spark off a wonderful idea or give you a solution to nagging problem?
+- Did you use someone else’s code?
+The app was built by our team we did't use someone else's code.
+
+- Do you want to thank someone explicitly?
+Yes all thanks to our team for puttings heads together to acheive this.
+
+- Did someone’s blog post spark off a wonderful idea or give you a solution to nagging problem?
 
 It's powerful to always give credit.
```

Comment:
If no blog post or someone's code to acknowledge, you can just acknowledge the team effort. Also ask @Theshedman and @anje123  if they are anybody they would like to acknowledge went carrying out their task